components/filter/filter-overflow-group.js

```diff
@@ -0,0 +1,26 @@
+import { css, LitElement } from 'lit';
+import { OverflowGroupMixin } from '../overflow-group/overflow-group-mixin.js';
+
+/**
+ * A component that can be used to display a group of filters that will be put into an overflow filter when they no longer fit on the first line of their container
+ * @slot - d2l-filters to be added to the container
+*/
+class FilterOverflowGroup extends OverflowGroupMixin(LitElement) {
+
+	static get styles() {
+		return [super.styles, css`
+			::slotted(d2l-filter) {
+				margin-right: 0.3rem;
+			}
+		`];
+	}
+
+	convertToOverflowItem() {
+	}
+
+	getOverflowContainer() {
+	}
```

Comment:
Are these required to be set for the overflow mixin to work?  Just wondering if there was any discussion in that PR about defining them in the mixin and throwing an error, like https://github.com/BrightspaceUI/core/blob/main/components/form/form-mixin.js#L61-L71.  Seems like sometimes we do that, sometimes we don't.

Reply:
There wasn't any discussion about that but that's a good idea in order to provide more information.